spec/controllers/books_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+describe BooksController do
+  describe '#index - GET/books' do
+    it 'returns a list of all books' do
+      books = create_list(:book, 5)
+
+      get :index
+      expect(response).to have_http_status :success
+    end
+  end
+
+  describe '#create - POST/book' do
+    it 'redirects to books#index after successfully saving the newly created
+    book to the database' do
+
+      book_params = {
+        book: {
+          title: "How to Bake Everything",
+          author: "Mark Bittman"
+        }
+      }
+
+      post :create, params: book_params
+      expect(response).to have_http_status :redirect
+    end
+  end
+end
```

Comment:
I'm glad you practiced writing this, but I think you can remove this whole test. Since you have a feature/acceptance test this is mostly duplicating what the other tests are doing. This will slow the test suite down without really giving you much more security